try {
} catch (Exception e) {
    var error = e;
}